src/run.rs

```diff
@@ -100,7 +100,7 @@ fn start_server_runtime(df: &Py<Dataflow>) -> PyResult<Runtime> {
 ///       `bytewax.recovery`. If `None`, state will not be
 ///       persisted.
 ///
-#[pyfunction(flow, "*", epoch_interval = "None", recovery_config = "None")]
+#[pyfunction]
 #[pyo3(text_signature = "(flow, *, epoch_interval, recovery_config)")]
 pub(crate) fn run_main(
```

Comment:
So, the `text_signature` here implies that `epoch_interval` and `recovery_config` are mandatory kwargs, while they are optional kwargs. Also, since `text_signature` doesn't generate ` signature`, while the reverse is true, if we want to show the default as "None" rather than "...", we need to use both `signature`  and `text_signature` ( https://pyo3.rs/v0.18.3/function/signature#overriding-the-generated-signature ):

```rust
#[pyo3(
    signature = (flow, *, epoch_interval = None, recovery_config = None),
    text_signature = "(flow, *, epoch_interval = None, recovery_config = None)"
)]
```